src/Maestro/Maestro.Common/AzureDevOpsTokens/AzureDevOpsTokenProvider.cs

```diff
@@ -111,6 +112,13 @@ private TokenCredential GetCredential(string account)
             var account = pair.Key;
             var option = pair.Value;
 
+            // 0. Use localy-cached credentials for development flows
```

Comment:
I'd say PAT should go first, then local credentials

Reply:
Sure, will change